src/lib/webpack-config.js

```diff
@@ -201,6 +201,36 @@ export default env => {
 			}
 		}),
 
+		customConfig({
+			module: {
+				loaders: [
+					{
+						enforce: 'pre',
+						test: /\.tsx?$/,
+						use: [
+							{
+								loader: resolve(__dirname, './npm-install-loader'),
+								options: {
+									modules: ['typescript', 'awesome-typescript-loader'],
+									save: true
+								}
+							},
+							{
+								loader: 'awesome-typescript-loader',
+								options: {
+									sourceMap: true,
+									useBabel: true,
+									babelOptions: createBabelConfig(env),
+									useCache: true,
+									configFileName: resolve(__dirname, 'ts-config.js'),
```

Comment:
I know that it is the standard name in the typescript ecosystem, same as with `.babelrc` for babel. Reason I went with this naming scheme is to be consistent with the other files in `preact-cli´. There isn't a `.babelrc` either, but `babel-config.js` instead. So I went with `ts-config.js`.

Plan is to use a user defined `tsconfig.json` if it is present at the projects root.